code/modules/mob/mob_verbs.dm

```diff
@@ -96,36 +96,84 @@
 	set name = "EORD Respawn"
 	set category = "OOC"
 
+	var/mob/living/liver
 	if(isliving(usr))
-		var/mob/living/liver = usr
+		liver = usr
 		if(liver.health >= liver.health_threshold_crit)
 			to_chat(src, "You can only use this when you're dead or crit.")
 			return
 
+	if(liver)
+		do_eord_respawn(liver)
```

Comment:
You could achieve the same result by doing
	if(!isliving(usr))
        return
	var/mob/living/liver = usr

Reply:
side note this appears to break respawning as ghost